src/lambda_streams/Animals.java

```diff
@@ -32,31 +34,54 @@ public static void main(String[] args) {
 
```

Comment:
There's no attempt in main, or the methods below, to implement the lambdas.  Expected lambdas matching the methods below to be created, and the methods below to be refactored to take lambdas as arguments.